test/ui-test/lightspeedUiTestExplorer.ts

```diff
@@ -0,0 +1,250 @@
+import {
+  ActionsControl,
+  ActivityBar,
+  By,
+  ContextMenu,
+  EditorView,
+  InputBox,
+  ModalDialog,
+  SideBarView,
+  VSBrowser,
+  ViewControl,
+  ViewSection,
+  WebView,
+  WebviewView,
+  Workbench,
+} from "vscode-extension-tester";
+import {
+  getFixturePath,
+  expectNotification,
+  getModalDialogAndMessage,
+  getWebviewByLocator,
+  sleep,
+  updateSettings,
+  workbenchExecuteCommand,
+} from "./uiTestHelper";
+import { expect } from "chai";
+import axios from "axios";
+
+export function lightspeedUIExplorerTest(): void {
+  describe("Test Lightspeed Explorer features", () => {
+    let workbench: Workbench;
+    let explorerView: WebviewView;
+    let modalDialog: ModalDialog;
+    let dialogMessage: string;
+    let sideBar: SideBarView;
+    let view: ViewControl;
+    let adtView: ViewSection;
+
+    before(async () => {
```

Comment:
It's a shame we need so much "start-up" and "tear-down" code to run the Explorer tests.